R/build_one.R

```diff
@@ -9,7 +9,7 @@ local({
 
   # input/output filenames are passed as two additional arguments to Rscript
   a = commandArgs(TRUE)
-  d = gsub('^_|[.][a-zA-Z]+$', '', a[1])
+  d = gsub('^./_|[.][a-zA-Z]+$', '', a[1])
```

Comment:
`.` is a special character (which matches any character), so you need to either escape it or put it in `[]`, e.g.

```suggestion
  d = gsub('^[.]/_|[.][a-zA-Z]+$', '', a[1])
```

Reply:
Sorry, sloppy on my part. Done!